cura/Machines/Models/DiscoveredPrintersModel.py

```diff
@@ -62,17 +62,28 @@ def readableMachineType(self) -> str:
         from cura.CuraApplication import CuraApplication
         readable_type = CuraApplication.getInstance().getMachineManager().getMachineTypeNameFromId(self._machine_type)
         if not readable_type:
-            readable_type = catalog.i18nc("@label", "Unknown")
+            readable_type = "unknown"
```

Comment:
Ah, but this should be human readable, and contrary to popular belief the French are also humans. I think what you intended to do is to have this properly translated, but not check against the translated texts later on. So I'd like you to:
* Revert this change.
* in `isUnknownMachineType`, check against `getMachineTypeNameFromId` to see whether it returns `None` instead of checking against the human-readable text.

Reply:
Done in ce5682aa7